zerver/lib/bulk_create.py

```diff
@@ -116,6 +118,26 @@ def bulk_create_users(
 
     Subscription.objects.bulk_create(subscriptions_to_create)
 
+    full_members_system_group = UserGroup.objects.get(
+        name="@role:fullmembers", realm=realm, is_system_group=True
```

Comment:
Should these `"@role:fullmembers"` etc. string be deduplicated and instead be centrally  hard-coded in `UserGroup.FULL_MEMBERS` etc. or something like that? There's some other places where these strings are repeated. Or is that intended since maybe this popped up in the reviews of the original implementation?

Reply:
I don't think we discussed in the original implementation, but I liked the idea of deduplicating this, does `UserGroup.FULL_MEMBERS_GROUP_NAME` sounds good. For other cases we can probably use `UserGroup.SYSTEM_USER_GROUP_ROLE_MAP`, I know this would be long but we already declare name of user groups there.

Reply:
Yeah, that name sounds good to me

Reply:
Yeah those names sound great.

Reply:
Reading this, I think the `UserGroup.SYSTEM_USER_GROUP_ROLE_MAP` references feel cumbersome. I think it's pretty reasonable to define constants like `EVERYONE_ON_INTERNET_GROUP_NAME = "@role:internet"` for all 5-6 of them, and have `SYSTEM_USER_GROUP_ROLE_MAP` use those values in its definitions, rather than this approach.

I'll drop the last commit and also tweak the other two final commits to fit that proposed pattern.